metasploit-resource-portal/source/_topnav.erb

```diff
@@ -1,9 +1,8 @@
 <% nav as: :pills, position: :static do %>
-  <%= link_to 'Docs',      'https://help.metasploit.com' %>
-  <%= link_to 'API',       'https://rapid7.github.io/metasploit-framework/api/' %>
-  <%= link_to 'Community', 'https://community.rapid7.com/community/metasploit' %>
```

Comment:
Just wanted to confirm this community link was intentionally removed before landing :+1:

Current production:

<img width="1098" alt="image" src="https://user-images.githubusercontent.com/60357436/167962761-599227e9-3cf6-4d88-8844-8c5026eb0f64.png">

Reply:
Yes that site was removed a few years ago.